agent/tool-scripts/kvm-spinlock

```diff
@@ -235,6 +257,9 @@ case "$tool" in
 	rabbit)
 	tool_cmd="$script_path/datalog/$tool-datalog $interval $username $password $tool_stdout_file"
 	;;
+	watches-cluster-health|watches-cluster-stats|watches-nodes-stats)
+	tool_cmd="$script_path/datalog/$tool-datalog $interval $tool_stdout_file $url $sniff $buffer $username $password $http_header_username $http_header_password $http_header_token"
```

Comment:
The fact that we have one line here supports the notion that we should have one script passing the appropriate parameter to get the desired behavior.